Bio/SeqIO/InsdcIO.py

```diff
@@ -1519,6 +1519,21 @@ class ImgtWriter(EmblWriter):
     FEATURE_HEADER = "FH   Key                 Location/Qualifiers\nFH\n"
 
 
+def convert(in_stream, in_format, out_stream, out_format):
```

Comment:
If you want this to be private (as per the docstring comment and prior usage), it needs a leading underscore.

Reply:
It cannot be private; this function is called from `SeqIO/__init__.py`

Reply:
The public function ``Bio.SeqIO.convert(...)`` can call a private function ``Bio.SeqIO.InsdcIO._convert(...)`` - that isn't a problem.